examples/src/main/java/tachyon/examples/Performance.java

```diff
@@ -228,18 +227,17 @@ public TachyonReadWorker(int id, int left, int right, ByteBuffer buf) throws IOE
     }
 
     public void readPartition() throws IOException {
-      TachyonByteBuffer buf;
       if (sDebugMode) {
+        ByteBuffer buf = ByteBuffer.allocateDirect((int) sBlockSizeBytes);
```

Comment:
Its a bit weird for the debug version to be performance sensitive, and it seems like we use a mix of direct/heap bytebuffers, but we can address this in another PR.